h2o-core/src/main/java/water/MRTask.java

```diff
@@ -386,14 +386,58 @@ private StringBuilder print(StringBuilder sb, int d) {
   public final T doAll( Vec vec, boolean run_local ) { return doAll(null,vec, run_local); }
   public final T doAll(byte[] types, Vec vec, boolean run_local ) { return doAll(types,new Frame(vec), run_local); }
 
-  /** Invokes the map/reduce computation over the given Frame.  This call is
-   *  blocking.  */
+  /** Invokes the map/reduce computation over the given Frame. This call is
```

Comment:
since you are fixing the comments, should we also have the first /** line empty, per javadoc convention?

Reply:
I will do it. :+1:

Reply:
fixed